uco-victim/victim.ttl

```diff
@@ -3,10 +3,11 @@
 # imports: http://unifiedcyberontology.org/role
 
 @base <http://unifiedcyberontology.org/victim> .
+@prefix : <http://unifiedcyberontology.org/victim#> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix victim: <http://unifiedcyberontology.org/victim#> .
```

Comment:
The only change I see in this file change is the removal of the "victim:" prefix.
As a standard practice each ttl file in the model should have its own prefix defined for clarity and should not use null prefixes for default base namespace.

Reply:
Because style conventions such as" each file should have its own prefix defined for clarity" can vary from developer to developer. I have used rdf-toolkit to normalize style conventions like this.
If we don't establish a means to automate style conformance merges are a nightmare.

Reply:
I agree that we should use consistent conventions.
The current convention is clearly that each ttl file should define and utilize its own explicit prefix. This was discussed and decided when UCO was created.
I have no objections to using rdf-tollkit to normalize these conventions but it should be normalizing on the decided convention, not changing it. Any such changes need to be explicitly agreed on and should not be part of this branch/PR.

Reply:
If the agreed upon conventions are documented that would be helpful. If they are not, it is a hindrance to making timely progress as I run up against undocumented convention that may or may not be industry best practices.

If conventions have not been documented, we need to get them documented. That will be problematic if the knowledge holder of the conventions is not available.

Reply:
Most conventions are not currently documented and I agree they should be.
That being said, the default should not be to attempt to change any current conventions without discussion and consensus.

Reply:
I suggested a bi-weekly or monthly call to iterate through proposed changes.  I think the OpenC2 SCs have a fairly agile approach where the proposed changes are tee'd up in bite size chunks.  All come prepared for the meeting and we close on everything we can.  Should help us quickly get through the low hanging fruit if we all do our homework and come prepared?

Reply:
The difficulty is that although I can agree in principle, I can't exercise the practice because I don't know the conventions that have been agree upon. Additionally, I am not certain that rdf-toolkit will accommodate us in all cases. 

I suggest that we start by updating the wiki with known conventions (and known issues with the convention violation)

When anyone sees a convention violation they can write an issue and I'll try to resolve it.

Reply:
@packet-rat I think that your suggested bi-weekly meeting is a good suggestion. We should do that. I am hoping to put a bit of "elbow-grease" into removing the non-substantive changes.
As far as organizing the meetings and the schedule I am resource tapped. Someone may have the time and energy to make your meetings happen, but sadly, I'm pushing my envelope as it is.
:-)